primitives/Cargo.toml

```diff
@@ -25,7 +25,7 @@ ark-pallas = "0.4.0"
 ark-poly = "0.4.0"
 ark-serialize = "0.4.0"
 ark-std = { version = "0.4.0", default-features = false }
-blst = { git = "https://github.com/EspressoSystems/blst.git", branch = "no-std", default-features = false } # TODO: pin to a tag or commit
+blst = { version = "0.3.11", default-features = false }
```

Comment:
This was originally changed to our fork in #254 . Prior to that it was
```toml
blst = "0.3.10"
```
@alxiong any reason for this switch? Any reason not to revert it now?

Reply:
Probably for `no_std`. IIRC we have CI for it